### ISSUE TITLE
Is this code base still active?

### ISSUE THREAD

**Issue by j-d-carmichael** [scionoftech/webp-converter#27]
6 months since the last commit. Is this still an active project or did @scionoftech get lucky and win a jackpot and now lives on their on island in the sun? :)

**Comment by scionoftech** [comment]
Yes — it's active again 🙂. **3.0.0** is a full modernization (TypeScript, command-injection fix, libwebp updated to 1.6.0, docs + tests). Closing.
